samples/SkiaMultiplatformSample/gradle.properties

```diff
@@ -1,4 +1,4 @@
 kotlin.code.style=official
-kotlin.mpp.enableGranularSourceSetsMetadata=true
-kotlin.native.enableDependencyPropagation=false
 org.gradle.jvmargs=-Xmx3G -XX:MaxMetaspaceSize=512m
+kotlin.version=1.9.20
+#skiko.composite.build=1
```

Comment:
Do we need these comments also here?

Btw, just want to know about our code style: do we need to add empty line at the end of each file or not or do not care about it ?